workflow-templates/publish.yaml

```diff
@@ -0,0 +1,14 @@
+# A CI configuration to auto-publish pub packages.
+
+name: Publish
+
+on:
+  pull_request:
+    branches: [ $default-branch ]
+  push:
+    tags: [ 'v[0-9]+.[0-9]+.[0-9]+*' ]
```

Comment:
Did we decide on using the `v` prefix?

Reply:
Yes, from a chat conversation (and from my pov, consistency with what we use for mono-repo tags).